.github/actions/setup-languages/action.yml

```diff
@@ -0,0 +1,20 @@
+name: Set up languages
```

Comment:
Replaced the `setup-database` script with this one. Now each workflow only has to call one setup action. `setup-project` if a db is needed. `setup-languages` if no db is needed.

Reply:
love it